### PR TITLE
Sic codes

### PR DESCRIPTION
**Pull request by jaleszek**
https://podio.com/bitzesty/app-qae/apps/stories/items/142

Implementation of sic codes storing and retrieving.

### CODE REVIEW

File: app/assets/stylesheets/admin/page-applications.scss [hunk 1]
```diff
@@ -241,3 +241,7 @@
 .section-applicant-details .text-right a {
   font-size: 14px;
 }
+
+span.centered {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Avoid qualifying class selectors with an element.

File: app/models/form_answer.rb [hunk 2]
```diff
@@ -52,6 +52,7 @@ class FormAnswer < ActiveRecord::Base
                              in: POSSIBLE_AWARDS
                            }
     validates_uniqueness_of :urn, allow_nil: true, allow_blank: true
+    validates :sic_code, format: { with: SICCode::REGEX }, allow_nil: true
```
Comment (matthewford) [inline app/models/form_answer.rb]: Inconsistent indentation detected.

File: spec/decorators/form_answer_decorator_spec.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
+require "rails_helper"
+
+describe FormAnswerDecorator do
+  describe "#average_growth_for" do
+    subject{ described_class.new form_answer }
```
Comment (matthewford) [inline spec/decorators/form_answer_decorator_spec.rb]: Space missing to the left of {.

File: lib/sic_code.rb [hunk 4]
```diff
@@ -0,0 +1,39 @@
+require "csv"
+
+class SICCode < ActiveYaml::Base
+  REGEX = /\A\d{4}(\/\d{1})?\z/ # based on the sic codes spreadsheet
+
+  set_root_path Rails.root
```
Comment (RuslanHamidullin) [inline lib/sic_code.rb]: Minor thing:
Not sure about best place for sic_codes.csv.
Maybe better place will be in db/fixtures/sic_codes.csv or config/sic_codes.csv.

**Comment by RuslanHamidullin** [comment]
Looks good for me :+1:

File: lib/sic_code.rb (hunk 4 above)
Comment (RuslanHamidullin) [inline lib/sic_code.rb]: ahh, @jaleszek ignore my previous comment as sic_codes.csv in .gitignore

Comment (dreamfall) [inline lib/sic_code.rb]: I agree.

Comment (jaleszek) [inline lib/sic_code.rb]: I haven't found the best place so left as it is...but I like idea of db/fixtures. will do

Comment (jaleszek) [inline lib/sic_code.rb]: Tbh. I'm thinking about making it a kind of convention (something-to-discuss-in-the-team-alert).

File: app/models/form_answer.rb (hunk 2 above)
Comment (matthewford) [inline app/models/form_answer.rb]: Inconsistent indentation detected.

**Pull request merged**